packages/manager/cypress/e2e/core/cloudpulse/alert-show-details.spec.ts

```diff
@@ -0,0 +1,241 @@
+const alertDetails = alertFactory.build({
+  service_type: 'dbaas',
+  severity: 1,
+  status: 'enabled',
+  type: 'system',
+  entity_ids: ['1', '2'],
+  rule_criteria: { rules: alertRulesFactory.buildList(2) },
+});
+const {
+  service_type,
+  severity,
+  rule_criteria,
+  id,
+  label,
+  description,
+  created_by,
+  updated,
+} = alertDetails;
+const { rules } = rule_criteria;
+const regions = [
+  regionFactory.build({
+    capabilities: ['Managed Databases'],
+    id: 'us-ord',
+    label: 'Chicago, IL',
+  }),
+  regionFactory.build({
+    capabilities: ['Managed Databases'],
+    id: 'us-east',
+    label: 'US, Newark',
+  }),
+];
+
+/**
+ * Integration tests for the CloudPulse DBaaS Alerts Detail Page, ensuring that the alert details, criteria, and resource information are correctly displayed and validated, including various fields like name, description, status, severity, and trigger conditions.
+ */
+
+describe('Integration Tests for Dbaas Alert Show Detail Page', () => {
+  beforeEach(() => {
+    mockAppendFeatureFlags(flags);
+    mockGetAccount(mockAccount);
+    mockGetRegions(regions);
+    mockGetAllAlertDefinitions([alertDetails]).as('getAlertDefinitionsList');
+    mockGetAlertDefinitions(service_type, id, alertDetails).as(
+      'getDBaaSAlertDefinitions'
+    );
+
+    cy.visitWithLogin('/monitor/alerts/definitions');
+
+    cy.wait('@getAlertDefinitionsList');
+
+    cy.get(`[data-qa-alert-cell="${id}"]`).within(() => {
+      // Ensure the alert label is visible
+      cy.findByText(label).should('be.visible');
+
+      // Action button within the alert cell should be visible and clickable
+      cy.get(`[data-qa-alert-action-cell="alert_${id}"]`)
+        .find('button')
+        .should('be.visible')
+        .click();
+    });
+
+    cy.get('[data-qa-action-menu-item="Show Details"]')
+      .should('be.visible')
+      .click();
+
+    cy.wait(['@getDBaaSAlertDefinitions']);
+  });
+
+  it('should correctly display the details of the DBaaS alert in the alert details view', () => {
+    // Validating contents of Overview Section
+    cy.get('[data-qa-section="Overview"]').within(() => {
+      // Validate Name field
+      cy.get('[data-qa-item="Name"]').within(() => {
+        cy.findByText('Name:').should('be.visible');
+        cy.findByText(label).should('be.visible');
+      });
+      // Validate Description field
+      cy.get('[data-qa-item="Description"]').within(() => {
+        cy.findByText('Description:').should('be.visible');
+        cy.findByText(description).should('be.visible');
+      });
+
+      // Validate Status field
+      cy.get('[data-qa-item="Status"]').within(() => {
+        cy.findByText('Status:').should('be.visible');
+        cy.findByText('Enabled').should('be.visible');
+      });
+
+      cy.get('[data-qa-item="Severity"]').within(() => {
+        cy.findByText('Severity:').should('be.visible');
+        cy.findByText(severityMap[severity]).should('be.visible');
+      });
```

Comment:
I think the test would still work if you had something just like this - could you verify? (not needing the within)
```suggestion
cy.findByText('Severity:').should('be.visible');
cy.findByText(severityMap[severity]).should('be.visible');
```

Reply:
@coliu-akamai , Just for understanding

Suggestion here is to directly use cy.findByText('Severity:').should('be.visible') , which finds the text in the entire Overview section but not in the severity group of the overview section, but we need to verify that this severity and value of the severity is in a single group and not in anywhere of the section, at that time using within and checking whether the label and value(text) is in severity group makes sense for me.

Let me know your thoughts on this